### ISSUE TITLE
Request: Please Add Signal To F-Droid

### ISSUE THREAD

**Issue by 0pLuS0** [signalapp/Signal-Android#7988]
Hello,

I thought Signal would be on F-Droid, please add it.

Thanks

**Comment by PiCob** [comment]
Actually not worth an answer since you didn't bother just deleting the issue template without even reading it... anyway: https://community.signalusers.org/search?q=F-droid

Please close this issue.

**Comment by 0pLuS0** [comment]
@PiCob, sorry, since I didn't bother reading? And where was I suppose to of read, or known about this?

**Comment by 0pLuS0** [comment]
@PiCob, sorry, since I didn't bother reading? And where was I suppose to of read, or known about this?

Please don't make assumptions that people know.

For your information, I only just started looking at Signal a few days ago, also the Community link you posted, doesn't show anything...

**Comment by PiCob** [comment]
When creating a new issue, the text box is pre-filled with this text template:



- [ ] I have searched open and closed issues for duplicates
- [ ] I am submitting a bug report for existing functionality that does not work as intended
- [ ] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.

### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)

### Screenshots



### Device info

**Device:** Manufacturer Model XVI
**Android version:** 0.0.0
**Signal version:** 0.0.0

### Link to debug log

**Comment by 0pLuS0** [comment]
@PiCob I know what GitHub is, I'm not new to GitHub.

By the way, there was no template that appeared when I was making this issue, nothing but blank white space...

Anyhow, I was only making a request, people do it all the time on GitHub and OpenSource...

**Comment by 0pLuS0** [comment]
@PiCob I know what GitHub is, I'm not new to GitHub.

By the way, there was no template that appeared when I was making this issue, nothing but blank white space...

Anyhow, I was only making a request, if possible, and if you have a link to this problem, why it isn't happening, then I'd appreciate seeing it, that community link shows nothing that I can see...

Thanks

**Comment by scienmind** [comment]
@0pLuS0 : the github issue tracker is used here to track bugs only.

Feature requests, questions, discussions and **anything else that isn't exactly a breakage in the intended application's functionality is to be directed to the forum**.

It is noted both in the issue template, and in repo's contributing.md guidelines, that's what piCob referred to.

**Comment by PiCob** [comment]
I see your point, but feature requests for Signal are tracked in the forums (link above) and that's what the issue template says as well (this is not**my** decision, but that of the Signal devs, and they close all feature requests and discussions anyway). Do you happen to use a third party software for GitHub manager/client? If not, please be so kind and create a topic in there forums describing how in detail you created the issue without seeing the issue template. If you use a third party within manager/client, you might post an issue to your GitHub client's tracker that it does not support GitHub issue templates.

However, here are some links to background explanations and the decision OWS have made:
https://community.signalusers.org/t/how-to-get-signal-apks-outside-of-the-google-play-store/808
https://github.com/signalapp/Signal-Android/issues/127
https://github.com/signalapp/Signal-Android/issues/281

I hope that helps. If you feel this topics needs more discussion, feel free to open a thread in the forums (https://community.signalusers.org).

**Comment by 0pLuS0** [comment]
@scienmind I understand you have your ways, everyone is entitled to do what they want...

I've been on GitHub for as long as it's been around, and in all those years, this is the first time someone has asked me to make a request on a forum.

I'm also not a casual GitHub user, I've been pretty much on GitHub every other day or week since it's been around, working on many projects, and helping many developers, I can't even begin to tell you how much, it's so massive. The fact is the majority are not sending people to forums to make requests.

If you've been around GitHub, then you know what I'm saying in regards to ```Requests``` made on GitHub projects...

```There's also something you and everyone else is very much missing the point on, that has gone over everyone's head. I really hope that you and whoever is in charge of this project is going to sit down and hear what I have to say, and seriously discuss this amongst yourselves.```

Now, when considered, online communication like this on GitHub or a Forum, online security is no difference. I first want to say, this is not what this topic is about. It's about ```Limting``` the users online presence, does that make sense? I am an extreme Security Geek, my online presence is very limited, and I do not visit forums, when I do it is extremely rare. I also do not visit any place online, in the manner of any such Community like gatherings you can think of, especially forums...

No there is not a Tin Foil Hat on top of my head, but as people developing a Security Based App, I would of thought that of all the people out there in Cyber Land, you would understand all of this.

Bottom line, to keep it simple, everyone's needs are different, some have a presence all over the Net and don't care, and for all the various reasons, some people are very limited in their online  presence, either way, respect needs to be given for both.

Plus I also personally don't view a Forum as a place for making Application Requests, because it's not the Point of Development, this is it, right here, so you might want to think about that. It should make it easier for you to keep track of such things here.

I hope I've made sense, and if I have, then respect those end-users like me that very much limit their on line presence. I certainly understand, chatting up a storm and asking questions here, no, take that to a forum. But making an application requests, yes, keep that here.

**Comment by scienmind** [comment]
@0pLuS0: Those ways aren't mine, they are signal-android development team's. I'm just another user, like most.

Over the years of app's development, handling of GitHub issues here was never smooth really. Unfortunately the dev team here is not in scale of vscode's one in terms of man power and resources. The team is small compared to app's user base. Over time the huge issue/request/etc submission rate became more than what the dev team was capable to handle. So several months ago they decided to declare "issue bankruptcy" of sorts, and accept only bug reports here moving forwards.

And yes, it's not perfect, and IMO, forum is a very poor platform for sorting/rating and managing feature request.

That being said, I think it's absolutely reasonable to respect moxie's team decision. Use the issue tracker for bugs and reduce the volume of distractions and notifications they're getting.

**Comment by 0pLuS0** [comment]
@scienmind, sorry I assumed you were DEV.

@greyson-signal, sorry I was just being the lazy Geek using GitHub like I have for years, as I've said, never been asked in all these years to go to a forum.

Anyhow after my digging, again was lazy and didn't look, I see there is a download on the site;

https://signal.org/android/apk/

But I'd rather not run around wasting anymore of my time, so can someone please point me as to the reason somewhere I can read, as to why it's not being put in F-Droid?

Thanks

**Comment by 0pLuS0** [comment]
@scienmind, sorry I assumed you were DEV.

@greyson-signal, sorry I was just being the lazy Geek using GitHub like I have for years, as I've said, never been asked in all these years to go to a forum.

Anyhow after my digging, again was lazy and didn't look, I see there is a download on the site;

https://signal.org/android/apk/

Can someone please point me as to the reason somewhere I can read, as to why it's not being put in F-Droid?

Thanks

**Comment by 0pLuS0** [comment]
@scienmind, sorry I assumed you were DEV.

@greyson-signal, sorry I was just being the lazy Geek using GitHub like I have for years, as I've said, never been asked in all these years to go to a forum.

Anyhow after my digging, again was lazy and didn't look, I see there is a download on the site;

https://signal.org/android/apk/

Can someone please point me as to somewhere I can read, as to why it's not being put in F-Droid?

Thanks

**Comment by Trolldemorted** [comment]
greyson has already posted the link for you, [here it is again](https://community.signalusers.org/t/signal-f-droid-repository/2808/7). You will also find links to the old discussions on this issue tracker there.

Why it is not on fdroid is simple: It is inferior to the website release, and the website release is there.

**Comment by 0pLuS0** [comment]
My bad for not digging through the forum post, but this seems to be the answer, right back here on GitHub... LOL

https://github.com/signalapp/Signal-Android/issues/127#issuecomment-13335689

Anyhow thanks...

**Comment by 0pLuS0** [comment]
My bad for not digging through the forum post, but this seems to be the answer, right back here on GitHub... LOL

https://github.com/signalapp/Signal-Android/issues/127#issuecomment-13335689

@moxie0 I hope you see this, because, as a Unix/Linux Geeks of many years, who doesn't wear a Tin Foil Cap, so no I truly don't believe everyone at Google is evil, but to say that you Care about the Security of your users, and then Distribute your APK through Google I find to be an Extreme Oxymoron.

Let's be real and honest here, since when, in the Real World of Extreme Cyber Computing, did anyone place their hands in the hands of Google? Please, this is absurd...

And since you have Snowden's words plastered on your site, I'd love to hear his take on Google...

Oh and by the way, I just read this is what Snowden thinks;

```People who care about their privacy should stay away from popular consumer Internet services like Dropbox, Facebook, and Google.```

Seems we have another Oxymoron in place...

Hmm
